build.gradle.kts

```diff
@@ -28,15 +28,15 @@ repositories {
 
 java {
     sourceCompatibility = JavaVersion.VERSION_11
-    targetCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_11
     withSourcesJar()
     withJavadocJar()
 }
 
 val projectVersion = project.property("version")?.takeIf { it != "unspecified" } ?: "1.0-SNAPSHOT"
 
 object Versions {
-    val ProvenanceProtos = "1.7.0-0.0.2"
+    val ProvenanceProtos = "v1.8.0-rc6"
```

Comment:
I talked to Ergels earlier about removing the `v` prefix from the version. You might want to check if that's still the case